pkg/mail/mail.go

```diff
@@ -25,7 +25,15 @@ import (
 	"k8s.io/release/pkg/log"
 )
 
-type MailSender struct {
+// GoogleGroup is a simple google group representation
+type GoogleGroup string
+
+const (
+	KubernetesAnnounceGoogleGroup     GoogleGroup = "kubernetes-announce"
```

Comment:
today we send to `kubernetes-announce` plus the `dev` do we want to keep that of just the `kubernetes-announce`?

Reply:
Yes, I added it to the list and changed the API function.